docs/web3.eth.account.rst

```diff
@@ -83,15 +83,13 @@ account you can just generate a random number that acts as a private key.
 
 To create a private key using web3.py and command line you can do:
 
-```shell
-python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.toHex(acc.privateKey)}, account={acc.address}')"
-```
+.. code-block:: shell
+
+    python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc.key)}, account={acc.address}')"
 
-Which outputs a new private key and an account pair:
+Which outputs a new private key and an account pair::
```

Comment:
```suggestion
Which outputs a new private key and an account pair:
```

Reply:
That's just to make the format of the following line into a 'literal block'. The backticks were not formatting very well.